projects/aws/image-builder/builder/types.go

```diff
@@ -119,7 +119,6 @@ type NutanixConfig struct {
 	ImageName         string `json:"image_name"`
 	ImageUrl          string `json:"image_url,omitempty"`
 	ImageExport       string `json:"image_export,omitempty"`
-	SourceImageName   string `json:"source_image_name,omitempty"`
```

Comment:
This was removed because we don't actually use it anywhere in upstream image-builder.